src/containers/header/menu.scss

```diff
@@ -0,0 +1,3 @@
+.mobileMenuActive {
+  display: flow-root !important;
```

Comment:
Doesn't it work without `!important`?
I knew that the `!important` is a bad practice.

Reply:
It doesn't work without !important